lms/djangoapps/api_manager/courses/views.py

```diff
@@ -366,6 +368,22 @@ def _get_course_data(request, course_key, course_descriptor, depth=0):
     data['resources'].append({'uri': resource_uri})
     return data
 
+def _get_static_tab_contents(request, course, tab, wrap_xmodule_display=True):
```

Comment:
`wrap_xmodule_display` argument is not need in this private method. It is always going to be `False` in this case.